src-cpp/app/modules/main/wallet/accounts/module.h

```diff
@@ -36,6 +26,15 @@ public slots:
 
 signals:
     void loaded() override;
+
+private:
+    void connect();
```

Comment:
oh please rename it cos `QObject::connect`. as a variant `void doConnect`. but not necessary :)